paper-server/src/main/java/org/bukkit/craftbukkit/entity/CraftAbstractArrow.java

```diff
@@ -68,12 +72,16 @@ public boolean isInBlock() {
 
     @Override
     public Block getAttachedBlock() {
+        return getAttachedBlocks().stream().findFirst().orElse(null);
```

Comment:
Complete nitpick, but if you wanna one-line this, just use Iterables.getFirst(getAttachedBlocks(), null).